.meta/links.toml

```diff
@@ -91,6 +91,8 @@ gzip = "https://www.gzip.org/"
 haproxy = "https://www.haproxy.org/"
 homebrew = "https://brew.sh/"
 homebrew_services = "https://github.com/Homebrew/homebrew-services"
+humio = "https://humio.com"
+humio_hec = "https://docs.humio.com/integrations/data-shippers/hec/"
```

Comment:
We are going to create a page in the docs just for Vector, let's just assume the uri is `s/hec/vector/`.

Reply:
Do we want to make this change now or once the page is up? It should be pretty trivial to add later as well.

Reply:
Let's wait until that page is up, incremental small fixes FTW.